protostar/commands/test/test_command.py

```diff
@@ -186,6 +192,7 @@ async def test(
         seed: Optional[int] = None,
         max_steps: Optional[int] = None,
         slowest_tests_to_report_count: int = 0,
+        gas_estimation_enabled: bool = False,
```

Comment:
why bother with `_enabled` since it's a boolean?

Reply:
We cannot, it is a problem with testing framework which destroys some information

Reply:
To make the name reflect the type. `gas_estimation` could be interpreted as an estimation result.